compiler/testData/codegen/bytecodeText/companion/kt14258_1.kt

```diff
@@ -3,7 +3,7 @@
 
 class My {
     companion object {
-        private val my: String = "OK"
+        val my: String = "OK"
```

Comment:
@pyos Please keep old test

Reply:
Then I'll have to mute it for JVM_IR, since a private const field's read is inlined. Is there any reason not to inline such fields that I've missed, by the way?

Reply:
Seems it's a bug in psi2ir or in some Lower, correspondingProperty/correspondingPropertySymbol should be always present //cc @dnpetrov

Reply:
@pyos This field could be leaked by @PublishedApi inline function, in another case it could be safely inlined

Reply:
@max-kammerer I found a bug in MoveCompanionObjectFieldsLowering: it does not set correspondingPropertySymbol on the new static backing fields.

Reply:
@pyos Good catch!